docs/modules/rust-guide/pages/rust-quickstart.adoc

```diff
@@ -121,43 +82,79 @@ cd rust_hello
 
 == Modify the default configuration
 
-Creating a new project adds a default `+dfx.json+` configuration file to your project directory.
-This file contains settings required to build a project for the {IC} much like the `+Cargo.toml+` file provides build and package management configuration details for Rust programs.
-You need to modify the default settings in both of these files to build Rust programs that run on the {IC} as canisters.
+Creating a new project adds several template files to your project directory much like when you create a new Rust package using the `+cargo new+` command.
+You need to modify these default files and add some Rust-specific files to your project before you can build canisters that run on the {IC}.
 
-=== Delete the default program
+=== Replace the default program
 
-Creating a new project creates a default `+src+` directory with a template `+main.mo+`. We'll eventually replace this with a `+lib.rs+` file. 
+Creating a new project creates a default `+src+` directory with a template `+main.mo+`. We'll replace this default program with a Rust library `+lib.rs+` package and `+Cargo.toml+` files. 
```

Comment:
```suggestion
Creating a new project creates a default `+src/rust_hello+` directory with a template `+main.mo+`. We'll replace this default program with a Rust library `+lib.rs+` package and `+Cargo.toml+` files. 
```

When I followed this tutorial, I started to get kind of confused around this part.

> Creating a new project creates a default `+src+` directory with a template `+main.mo+`.

When I read that, I expect that somewhere there is a file like `./src/main.mo`, but actually dfx (0.7.0) creates this file structure for me:

```
~/ic-projects/rust_hello  master ✔                                                                                        1m  
▶ tree src 
src
├── rust_hello
│   └── main.mo
└── rust_hello_assets
    ├── assets
    │   ├── logo.png
    │   ├── main.css
    │   └── sample-asset.txt
    └── src
        ├── index.html
        └── index.js
```

So the main.mo is nested not just within `src` but within `src/rust_hello`.